src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/AdjudicationHearingResult.kt

```diff
@@ -73,7 +73,7 @@ class AdjudicationHearingResult(
 
   /* Having to make optional as contains values from OIC_FINDING domain in older records.
      Mapping code directly in separate property */
-  @ManyToOne(fetch = FetchType.LAZY)
+  @ManyToOne
```

Comment:
LAZY was being ignored (and generated a warning) since this has a NotFound meaning it *must* be eager loaded